client/components/pie-chart/style.scss

```diff
@@ -0,0 +1,80 @@
+.pie-chart__chart-section-0 {
+	fill: $blue-dark;
+}
+
+.pie-chart__chart-section-1 {
+	fill: $blue-medium;
+}
+
+.pie-chart__chart-section-2 {
+	fill: $blue-light;
+}
+
+.pie-chart__legend-sample-0 {
+	fill: $blue-dark;
+}
+
+.pie-chart__legend-sample-1 {
+	fill: $blue-medium;
+}
+
+.pie-chart__legend-sample-2 {
+	fill: $blue-light;
+}
```

Comment:
I suspect chart chart sections and legend samples will always have the same color respectively. As such, we could remove duplicate code and group them like this:

```css
.pie-chart__chart-section-0,
.pie-chart__legend-sample-0 {
	fill: $blue-dark;
}

.pie-chart__chart-section-1,
.pie-chart__legend-sample-1 {
	fill: $blue-medium;
}

.pie-chart__chart-section-2,
.pie-chart__legend-sample-2 {
	fill: $blue-light;
}
```